examples/main/main.cpp

```diff
@@ -64,6 +64,7 @@ struct whisper_params {
     int32_t max_len      =  0;
     int32_t best_of      = whisper_full_default_params(WHISPER_SAMPLING_GREEDY).greedy.best_of;
     int32_t beam_size    = whisper_full_default_params(WHISPER_SAMPLING_BEAM_SEARCH).beam_search.beam_size;
+    int32_t audio_ctx = 1500;
```

Comment:
Default it to `0` - this way `whisper.cpp` will automatically use the correct value:

```suggestion
    int32_t audio_ctx   = 0;
```